openssh/files/sshd_config

```diff
@@ -128,8 +128,14 @@
 {{ option_default_uncommented('RSAAuthentication', 'yes') }}
 {{ option_default_uncommented('PubkeyAuthentication', 'yes') }}
 {{ option('AuthorizedKeysFile', '%h/.ssh/authorized_keys') }}
-{{ option('AuthorizedKeysCommand', 'none') }}
-{{ option('AuthorizedKeysCommandUser', 'nobody') }}
+{%- if server.authorized_keys_command is defined %}
+AuthorizedKeysCommand {{ server.authorized_keys_command.command }}
```

Comment:
I made two mistakes. (Correct me if I'm wrong.)

1st mistake:
As far as I understand it (now), the variable `server` never gets assigned in the whole formula. 

2nd mistake:
`option('AuthorizedKeysCommand...` is mandatory here, because it makes use of `map.jinja` and provides default values.

I'm sorry, I misled you.